src/components/Permissions/DataPermissions/DataPermissions.jsx

```diff
@@ -34,19 +28,20 @@ import ListItem from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItem'
 import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
 import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
 import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import { buildAppsQuery, buildKonnectorsQuery } from 'lib/queries'
+import { useQuery } from 'cozy-client'
 
 const DataPermissions = ({ t }) => {
   const { permission } = useParams()
 
-  const THIRTY_SECONDS = 30 * 1000
-  const queryResultApps = useQuery(Q(APPS_DOCTYPE), {
-    as: APPS_DOCTYPE,
-    fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
-  })
-  const queryResultKonnectors = useQuery(Q(KONNECTORS_DOCTYPE), {
-    as: KONNECTORS_DOCTYPE,
-    fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
-  })
+  const queryResultApps = useQuery(
+    buildAppsQuery().definition,
+    buildAppsQuery().options
```

Comment:
nit: maybe we can avoid duplicate `buildAppsQuery()` call here.
Hopefully those methods are not heavy to compute so it is not really a problem here, but as the method return a combined object it would be great to benefit from it (we can also split the method)

(same comment for all other `buildKonnectorsQuery`, `buildKonnectorsQueryBySlug` etc